G/GAP_pkg/GAP_pkg_juliainterface/build_tarballs.jl

```diff
@@ -9,23 +9,23 @@ using Pkg
 uuid = Base.UUID("a83860b7-747b-57cf-bf1f-3e79990d037f")
 delete!(Pkg.Types.get_last_stdlibs(v"1.6.3"), uuid)
 
-gap_version = v"400.1200.200"
-gap_lib_version = v"400.1201.200"
+gap_version = v"400.1300.0"
+gap_lib_version = v"400.1300.0"
 name = "JuliaInterface"
-upstream_version = "0.8.3" # when you increment this, reset offset to v"0.0.0"
+upstream_version = "0.9.0" # when you increment this, reset offset to v"0.0.0"
 offset = v"0.0.0" # increment this when rebuilding with unchanged upstream_version, e.g. gap_version changes
 version = offset_version(upstream_version, offset)
 
 # Collection of sources required to build this JLL
 sources = [
-    GitSource("https://github.com/oscar-system/GAP.jl", "38af454942a1924b39f30f00a8a64622dd06ce32"),
+    GitSource("https://github.com/oscar-system/GAP.jl", "bc4b5c2374299564184fb79f00fedbba4a255d39"),
 ]
 
 # Bash recipe for building across all platforms
 script = raw"""
 cd GAP.jl/pkg/JuliaInterface
 ./configure --with-gaproot=${prefix}/lib/gap
-make -j${nproc} CFLAGS="-I${includedir} -I${includedir}/julia" LDFLAGS="-ljulia"
+make CFLAGS="-I${includedir} -I${includedir}/julia" LDFLAGS="-ljulia -lgap" V=1
```

Comment:
Adding `-lgap` here is a hacky workaround. I need to fix this properly in `GAP_jll`, but for now it is good enough to let me progress other bits.